app/dashboard/migrations/0127_merge_20181207_1132.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.2 on 2018-12-07 11:32
```

Comment:
All of these migrations need deleted and recreated as a single migration **following** updating your fork branch with `master`